develop/tutorials/articles/100-tooling/04-liferay-workspace/05-managing-the-target-platform-for-liferay-workspace.markdown

```diff
@@ -0,0 +1,153 @@
+# Managing the Target Platform for Liferay Workspace
+
+Liferay Workspace provides a way to target a specific release of @product@ to
+develop against. This greatly decreases the need to manage dependencies and
+their versions when developing your app. For example, suppose you've developed
+your app for the current release of @product@, but then a new version is
+released. Of course, many of the artifact versions have incremented, so you must
```

Comment:
`so in some cases, you may want to update your own app's dependencies to exactly match the artifacts in the version of the @product@ you are targeting.`

Reply:
@codyhoag for this sentence, we probabaly should have a callout to another doc where we describe the cases that
1) you don't want to track the dependency artifacts exactly 1:1
2) times you do want to track them 1:1

Reply:
@gamerson I'm not 100% certain for the cases where we'd track dependencies 1:1 vs not doing it 😟 . Could you list the cases for both scenarios? I can add that in to our docs.